docs/docs/groups-api/group-type-definitions.md

```diff
@@ -265,4 +273,15 @@ Used as an input for group modification endpoints (create, edit, add members, et
 }
 ```
 
+### `(Object)` Group Activity
+
+| Field     | Type                                                                  | Description                              |
+| :-------- | :-------------------------------------------------------------------- | :--------------------------------------- |
+| groupId   | integer                                                               | The group's ID.                          |
+| playerId  | integer                                                               | The players's ID.                        |
+| type      | [ActivityType](/groups-api/group-type-definitions#enum-activity-type) | The type of activity                     |
+| role      | [GroupRole](/groups-api/group-type-definitions#enum-group-role)?      | The player's role (rank) in the group.   |
+| createdAt | date                                                                  | The date at which the activity happened. |
+| player    | [Player](/players-api/player-type-definitions#object-player)          | The activity entry's parent player ID.   |
```

Comment:
This seems misleading since this is the player object, not their ID.

Recommend something like: The player associated with this activity.